src/main/java/net/minecraft/block/SlimeBlock.java

```diff
@@ -71,4 +76,15 @@ public void onEntityWalk(World worldIn, BlockPos pos, Entity entityIn)
 
         super.onEntityWalk(worldIn, pos, entityIn);
     }
+
+    @Override
+    public float getSlipperiness() {
+        if (noSlow().isEnabled2() && noSlow().getBooleanValueFromSettingName("Blocks"))
+            return 0.6f;
+        return super.getSlipperiness();
```

Comment:
```suggestion
        // MODIFICATION BEGIN: less slipperiness if NoSlow is enabled, this is the local player, and Blocks is enabled in NoSlow.
        if (noSlow().isEnabled2() && noSlow().getBooleanValueFromSettingName("Blocks"))
            return 0.6f;
        return super.getSlipperiness();
        // MODIFICATION END
```
also, why not 0.0f slipperiness?